frontend-cert/js-projects/fortune-teller-app/user-stories.md

```diff
@@ -0,0 +1,10 @@
+1. You should have a set of 5 fortunes as strings and assign them to different variables.
```

Comment:
```suggestion
1. You should declare five variables, each with a string value that describes a fortune.
```

set is a built-in object in JS, I think we should be a little more specific here.